### ISSUE TITLE
 As a user I would like to be able to clearly read the text on the application landing page that describes the different types of permits that are available to me so that I can more quickly and effectively chose a permit type.

### ISSUE THREAD

**Issue by DawnDesigner** [18F/fs-open-forest-platform#565]
## Notes
** As a user I would like to be able to clearly read the text on the application landing page that describes the different types of permits that are available to me so that I can more quickly and effectively chose a permit type.**

## Acceptance Criteria
- [x] The titles are clear and consistent and short.
- [x] The text does not overlay an image.
- [x]  Action buttons that take user to correct application page and are easy to see and understand.
- [x] The page is functional and easy to view on mobile.
- [x] Conduct User Testing

## Tasks
- [x] Create a design for two boxes split between text on a solid color field and the other side an image relevant to the forest
- [x] Complete layout in dev to look like mock up (No italics. Create bullet list for each description)
- [x] Dev for mobile
- [x] Make sure links work
- [x] PO approved

## Definition of Done
- [ ] Page looks like mock up

- [ ] All links work
- [ ]  508 is tested (alt text, color contrast and tabbing)
- [ ] Usability tested

![LandPgContenchanged](https://user-images.githubusercontent.com/47989077/54388400-02490000-4674-11e9-8079-23f04d449b44.png)

**Comment by DawnDesigner** [comment]
Amber comments from #533
"Content boxes, roll over text: this is interesting concept. can we do that w/508 compliance, and as a mobile app?"

Not sure on this, but there is no roll over text.  If you are referring to the two long boxes under the images, these are clickable accordions that slides open to reveal content that is commonly used, mobile friendly and 508.

(example: https://www.w3schools.com/howto/howto_js_accordion.asp )

**Comment by DawnDesigner** [comment]
Here is modified mock up after meetings and input.  @ASprinkle  and @MelissaBraxton -Here is the solution we came up with:

1. Boxes with text not over image and links to apply

2. For more information, an opportunity to compare permit types with a scroll bar to manage the size of the descriptions better on the page.

3. A link for the step through that already exists on the site.

  Please review the mock up and if you feel this is ready please check the PO approval box and it can go to the Ready column.

Thanks

![SpecialUse-LandingPg-ChoosePermit](https://user-images.githubusercontent.com/47989077/55909121-364b0e80-5ba9-11e9-96cc-319e90fb728a.png)

**Comment by jdamis4XD** [comment]
@ASprinkle @DawnDesigner @lisaredux @carlsonem  Since Dawn is out on training this week I did a little digging around to get some context on the issue.  The recommended design is a big departure from what is currently there and I am trying to confirm if that was driven by user feedback?  I am not sure having relevant content to help with the decision making process should be hidden in an accordion when there appears to be more space on the page and doesn't really fulfill the story need to clearly read the text.  It might make sense to do some additional testing around copy with users to determine what information needs to be on the screen to accommodate both desktop and mobile users if we don't already know.  This story along with #530 and #531 all require testing around copy, so I would recommend moving this story over to the backlog to give the design team an opportunity to review and rework.

**Comment by ASprinkle** [comment]
Keeping this story focused on the "text on the application landing page". Created another story to address the user flow regarding minimal clicks after moving away from the landing page.

**Comment by DawnDesigner** [comment]
Final mock up - Needs @ASprinkle  approval.  To address the question is this still in keeping with the users request, the answer is yes.  This iteration addresses only the request to remove the photos from behind the text so it's easier to read.

The previous mock up  was addressing a second user request to not have to click from this page to get more information about permit types.  The design team decided to break this into two stories so the flow of information could be discussed and resolved before giving further direction to the development team.  So for now, this is the solution to the most pressing issue the landing page has.

Once Amber gives this her nod, I'll supply the photos and graphic for the page.  I'm in the process now of getting permission in writing for the images.

![MtBaker Just Boxes](https://user-images.githubusercontent.com/47989077/57549655-24889280-7332-11e9-929b-b2fc1070e3f0.png)

**Comment by ASprinkle** [comment]
@DawnDesigner looks good. Let's move with this draft of the mock up. Thank you!

**Comment by DawnDesigner** [comment]
Contacted Public Services Staff Officer for Mt Baker - he will confirm what he suspects that there is no problem using the images.  

Mock ups to show mobile/tablet layout for responsive coming up.

**Comment by ASprinkle** [comment]
Moved to Blocked until we hear back from the MBS on approvals for using the photos.

**Comment by DawnDesigner** [comment]
Approval received to use images. These images are public use.

**Comment by DawnDesigner** [comment]
A/B testing with two prototypes in progress for these issues and will be removed from blocked when testing is completed and these issues are refined and put into ready.
#530 , #531, #565 and #627

**Comment by ASprinkle** [comment]
@DawnDesigner @lisaredux I had a good meeting with Sue from the MBS today. One of the things that was a high priority for her was to make sure we're using photos that portray the intent/purpose of the permit. For example: Non-Commercial Group Use permits should have a photo of a group (like in the exampled provided in the Learn More page) vs. a single person standing on a look out tower. Temporary Outfitting and Guiding permits should have a photo example of that (i.e., stock train, white water rafting, mountain climbing etc.) vs. a mountain scene. I think we can use the approved Flicker account and come up with some good options there.

**Comment by DawnDesigner** [comment]
@ASprinkle  Thanks Amber, already started the process of identifying new images that will work to her specifications.

**Comment by DawnDesigner** [comment]
@ASprinkle  - I recommend we move this to in progress. I've identified potential images, will mock them up and post them here at the end of the week.  Thanks!

**Comment by ASprinkle** [comment]
@DawnDesigner I thought this story was blocked along with 627, 530, 531, waiting on the results of the testing taking place this week(?)  Then, next sprint, these stories would be completed.

**Comment by DawnDesigner** [comment]
@ASprinkle  - @lisaredux  discussed and decided this is about the design/layout elements - and not about the content itself - which is US #627. So this story now needs two new images and then could go into the backlog for prioritizing. 

If you feel we need to wait until the actual copy is written (#627)  please advise. No worries. We can leave it blocked until we get that closed.

This testing study is now complete. Findings and recommendations can be found on this repositories Wiki (Design and Research) here:

https://github.com/18F/fs-open-forest-platform/wiki/Usability-Testing:-Special-Uses-(Content-Flow-Study)-June-2019

**Comment by ASprinkle** [comment]
Thanks @DawnDesigner

**Comment by ASprinkle** [comment]
@DawnDesigner As a follow up to the testing outcomes. https://github.com/18F/fs-open-forest/wiki/Usability-Testing-%E2%80%94-Special-Use-permits-evaluation-content-(June-2019)

I have added the following information to the tasks for this story.
Remove italics
Insert bullets

Agreed, content is addressed in 627

**Comment by DawnDesigner** [comment]
@ASprinkle  @carlsonem  Three images that are possible selections from the MT Baker-Snoqualmie flickr site and available for use. Please let me know if you think these work.  I wasn't certain if the one with Smokey and Hoot work because that might be seen as a FS event, so I chose one that looked more public facing.

These images are sized for production, so if they work we can move this into dev for staging.  I've already discussed the dimensions with development to make sure he has the right specs, and that it's within the template. It is ready for responsive.


![SU - Landing Page 3](https://user-images.githubusercontent.com/47989077/59945877-eacda000-9436-11e9-9afe-cb420bdfced0.png)
![MtBakerRiverSmokey](https://user-images.githubusercontent.com/47989077/59945889-f4570800-9436-11e9-90ed-f2acc2c416cb.jpg)
![MtBakerRiverRafting](https://user-images.githubusercontent.com/47989077/59945895-f751f880-9436-11e9-9925-12d3117c6992.jpg)
![MtBakerMtnSitting](https://user-images.githubusercontent.com/47989077/59945900-f9b45280-9436-11e9-9c52-f21aef4a0993.jpg)

**Comment by ASprinkle** [comment]
@DawnDesigner The photos you selected look great. I think they show exactly what the special uses administrator was asking for. I agree, do not use the smokey bear/woodsy photo because it may give the impression of an FS hosted event.

The mock design is easy to read, and looks like it meets the objectives. Let's hand it off to dev! :)

**Comment by carlsonem** [comment]
@mtlaney  this should be ready for development

**Comment by DawnDesigner** [comment]
@ASprinkle  - Thanks Amber.

**Comment by mtlaney** [comment]
@DawnDesigner 

**do we have the graphic for the following?**

<img width="668" alt="Screen Shot 2019-06-27 at 1 17 18 PM" src="https://user-images.githubusercontent.com/18233188/60286641-f2c98c00-98dd-11e9-95b5-a488dadfa169.png">

**Comment by lisaredux** [comment]
@mtlaney - we'll get this graphical element to you ASAP

**Comment by DawnDesigner** [comment]
Sorry, didn't see your comment.  On the way

**Comment by DawnDesigner** [comment]
Here is the final image Mikie, I optimized it to double the correct size for mobile and responsive.
Sorry - I must of deleted this notice by accident. Ping me on Teams if you need anything else in case I don't see the notification in the future.

![Tree Silhouette](https://user-images.githubusercontent.com/47989077/60620144-6e2cb100-9da8-11e9-8895-ef6bd13b4879.jpg)

**Comment by ASprinkle** [comment]
@mtlaney this looks good to me. I am unable to test it/review it on a mobile application. I think we can address that later if it's an issue.

**Comment by mtlaney** [comment]
@ASprinkle lets setup like 5 minutes tomorrow, I'll walk you thru how to view it on mobile devices 👍
